deploy/k8s/chart/templates/init-guac/020-Job.yaml

```diff
@@ -22,7 +22,7 @@ spec:
   ttlSecondsAfterFinished: 600
   template:
     spec:
-
+      serviceAccountName: clusters-service
```

Comment:
I am not sure this makes sense in our upstream environment. Please add this as a value in the values file.

Reply:
ok